std_assert_test.go

```diff
@@ -0,0 +1,238 @@
+package objx_test
```

Comment:
Add a comment that mentions why this file exists: this is a reimplementation of a subset of Testify's API to break a cycle dependency with Testify.

Reply:
i added some words about that